cmd/protoc-gen-go-grpc/main.go

```diff
@@ -32,13 +32,23 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"google.golang.org/grpc"
+	"os"
+	"path/filepath"
 
 	"google.golang.org/protobuf/compiler/protogen"
+
 )
 
 var requireUnimplemented *bool
 
 func main() {
+	if len(os.Args) == 2 && os.Args[1] == "--version" {
```

Comment:
What happens with `binary --version nope`?

For `gcc` it’s this:
```
$ gcc --version nope
gcc: error: nope: Datei oder Verzeichnis nicht gefunden
gcc (SUSE Linux) 7.5.0
```

Also, we have included the `flag` package which is how flags are expected to be be parsed and processed. Manually parsing the flag yourself seems unwise.

Reply:
Cool! Good point! Fixing it now.